src/styles/blocks/page.scss

```diff
@@ -0,0 +1,10 @@
+.page {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+}
```

Comment:
It seems like you have declared the font family 'Roboto' twice in both '.page' and '.page__body'. This might be unnecessary if '.page__body' is a child element of '.page'. If you want to apply the same font family to all elements within '.page', you can declare it once in '.page'. This is a code quality issue.